### PR TITLE
edited the Readme (is the linked image okay?)

### PR DESCRIPTION
**Pull request by trytomakeyouprivate**

### CODE REVIEW

**Review by iaacornus — COMMENTED** [review]
i'm not sure about uploading precompiled binary since for users, it can be seen as malicious or just not in a good way. we can just put a script that clones the repo and builds the project, or look if theres an already compiled bin in the repo for it

**Review by iaacornus — APPROVED** [review]

**Review by iaacornus — CHANGES_REQUESTED** [review]
i think we should not include precompiled binaries, and there are some claims that is quite far from truth

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
 A small program making the install of Fedora Silverblue / Kionite easy. It lets you choose what to install or set.
+
+## Why this is needed
+An OSTree-based Fedora system can result in a pretty perfect and unbreakable experience.
+- Your system stays pretty much the same as the developers version. This makes bugfixes and support easier.
+- You install external Apps as Flatpaks or through Podman containers. You can use any Flatpak, Ubuntu, Fedora, Arch e.g. apps you want, but they can't break your system
+- The system can update automatically, if an update breaks something, you can reboot and use the older version.
+
+But Fedora Kinoite/Silverblue (and possibly versions with other Desktops like XFCE) are not really well setup out of the box. 
+- no automatic updates for System and Flatpaks, system updates take very long though
```
[the remark this answers is not in the document]
Comment (trytomakeyouprivate) [reply README.md:12]: Hm that config is small. I havent found the correct systemd .timer yet.

tonywalkers script has a systemd timer with more configs, I  only see here how updates are handled, if they are applied live etc.

I would want configs for
- how often
- at what time
- what time span

**Comment by trytomakeyouprivate** [comment]
ok removed it, you are right. I mean including it doesnt mean its installed, but I just left if in the readme for now

**Review by iaacornus — APPROVED** [review]

**Comment by iaacornus** [comment]
you should have a permission over the repo now, since i will be quite busy for a while, but i will join you as soon as possible

File: README.md [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+- Native Firefox installed but it can't play Videos
+- adding COPR repositories or other external ones is complicated
+- changing the SDDM theme is not possible, but the standard theme looks very outdated
```
[the remark this answers is not in the document]
Comment (trytomakeyouprivate) [reply README.md:15]: I will edit that. Quite old, sorry

**Pull request merged**